apps/ledger-live-mobile/src/screens/Portfolio/Header.tsx

```diff
@@ -17,14 +17,29 @@ import { track } from "../../analytics";
 import useDynamicContent from "../../dynamicContent/dynamicContent";
```

Comment:
In this file we split the `Header` component in smaller components with their internal state (via the hooks they use) to avoid re-renders of the entire component for updates that are specific to one of its parts.